### ISSUE TITLE
Select Text and Clipboard

### ISSUE THREAD

**Issue by technatica** [leo-editor/leo-editor#218]
I use a spellcheck/thesaurus app called Artha (highly recommended) that can be invoked on selected text. In every other application on my Linux workstation (I tried 22 of them, including ones with no editing capability), you highlight the text and invoke Artha. Artha loads the highlighted text and provides its function.

If I highlight a word in Leo using the mouse, either via double-click or click and drag, Artha works as expected. If I select the word using Shift-Arrow key, as far as Artha is concerned the text doesn't exist. Hitting Ctrl-C before invoking Artha does nothing to change this, so it appears that it doesn't use the clipboard. 

This is getting annoying, as I use Leo as my primary writing tool, and having to pull my hands from the keyboard in order to select the word is getting old.

**Comment by edreamleo** [comment]
Thanks for the report.  I'll compare the two different ways of selecting text.  Presumably there is some difference.  Artha sounds promising.  I'll test any changes with it.

**Comment by tbnorth** [comment]
On Wed, 18 Nov 2015 08:31:37 -0800
"Edward K. Ream" <notifications@github.com> wrote:

> Thanks for the report.  I'll compare the two different ways of
> selecting text.  Presumably there is some difference.  Artha sounds
> promising.  I'll test any changes with it.

https://groups.google.com/forum/#!topic/leo-editor/Cj1vmPmniqs
analyses this problem, think it should be a straight forward fix.

Cheers -Terry

**Comment by edreamleo** [comment]
Fixed at 0c89aff.

**Comment by edreamleo** [comment]
Alas, this bug interacts with #241.  With PyQt5 the following reasonable code in LeoQtGui.setClipboardSelection generates a spurious warning message:

```Python
cb = self.qtApp.clipboard()
cb.setText(QString(s), mode=cb.Selection)
```

The workaround is not to execute this code with PyQt5, but in effect that reopens *this* bug.
